CMakeLists.txt

```diff
@@ -17,19 +17,26 @@ endif()
 set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
 set(BRAYNS_DESCRIPTION "Hardware Agnostic Ray-Tracer")
 
-option(BRAYNS_BRION_ENABLED "Activate Brion" OFF)
-option(BRAYNS_DEFLECT_ENABLED "Activate streaming to display wall" OFF)
-option(BRAYNS_NETWORKING_ENABLED "Activate networking interfaces" OFF)
-option(BRAYNS_ASSIMP_ENABLED "Activate assimp library for standard 3D file format management" OFF)
-option(BRAYNS_IMAGEMAGICK_ENABLED "Activate IMAGEMAGICK library for standard image file format management" OFF)
+include(Common)
+
+# By default, optional dependencies are disabled
+set(BRAYNS_DEPENDENCY_ENABLED OFF)
```

Comment:
We discussed the auto detection of libraries and this is something I agree this should be done. I'll do that in a later PR though. Thanks.